src/app/profile/page.tsx

```diff
@@ -0,0 +1,23 @@
+import FileInput from '@/components/pages/profile/file-input'
+import FormProfile from '@/components/pages/profile/form-profile'
+import Content from '@/components/ui/content'
+import { ThemeToggler } from '@/components/ui/theme-toggler'
+
+export default function Profile() {
+  return (
+    <section className="h-screen w-screen bg-gradient-blue-white">
+      <Content className="flex items-center justify-center dark:bg-none dark:bg-base-gray-8">
+        <div className="p-7 sm:p-10 flex gap-7 md:gap-[3.75rem] rounded-3xl bg-base-white dark:bg-base-black">
+          <FileInput />
```

Comment:
I would make a convention to name all things with common prefix. i.e. `InputFIle`, but as you wish, you can keep it

Reply:
Honestly, I'm fully disagree with u. All our components are named according to the next pattern:
Theme _Toggler_
Dropdown _Menu_
File _Input_
Theme _Provider_
e.c.

Reply:
You can name it as you want, but why this pattern, are there any advantages in this naming?

Reply:
Do I know? Default name pattern in shadcn, also in others frontend libraries(tailwind, bootstrap). Btw, are there any advantages in your naming(I mean for us, but don't mind to listen about general benefits too)?

Reply:
Very often files and folders are sorted in alphabetical order, so If we name them with common prefix it will look nice in fill tree 😊

Something like this 

<img width="427" alt="image" src="https://github.com/dev-KPI/messenger-frontend/assets/95881717/0bfea137-a07b-4b0d-bb20-5883e2b85b26">

Reply:
Sorry, disagree with u, I won't add shadcn component and then rename it(as rename a whole bunch of other components). But u could create issues and make it by yourself😊😊😊

Reply:
Oh, I forgot, u don't contribute to the frontend part 😭

Reply:
For now just let it be how it is, there is no such importance of renaming it yet